app.py

```
@@ -37,91 +26,66 @@ def get_wiki_pages():
-
 @app.route("/update", methods=["GET"])
 def update():
     title = request.args.get("title")
     wiki = request.args.get("wiki")
-    encoded_title = quote(title) if title else ""
-    url = f"https://{wiki}/wiki/{encoded_title}"
-
+    if not title or not wiki:
+        return render_template("result.html", result_class="danger", tt="title_or_wiki_required"), 400
     if wiki not in valid_wikis:
         return render_template("result.html", title=title, url=url, result_class="danger", tt="wiki_not_supported", tt1=wiki), 400
```

Ruff (0.8.2)

Undefined name `url`

(F821)